cameo/api/designer.py

```diff
@@ -170,7 +170,7 @@ def __call__(self, product='L-glutamate', hosts=HOSTS, database=None, aerobic=Tr
         notice("Starting searching for compound %s" % product)
         try:
             product = self.__translate_product_to_universal_reactions_model_metabolite(product, database)
-        except KeyError:
+        except Exception:
```

Comment:
Yes, I should be able to catch KeyError there. Unfortunately the underlaying implementation needs to be corrected (e.g. `cameo.api.products.products.search`) but there are many other exceptions that need to be refactored. In this case, any exception that comes from `__translate_product_to_universal_reactions_model_metabolite` result from not being able to find any metabolite (could also be index errors on pandas data frames).